src/perf/lib/Tcp.cpp

```diff
@@ -199,10 +199,15 @@ TcpWorker::~TcpWorker()
     CxPlatEventUninitialize(WakeEvent);
 }
 
-bool TcpWorker::Initialize(TcpEngine* _Engine)
+bool TcpWorker::Initialize(TcpEngine* _Engine, uint16_t)
 {
     Engine = _Engine;
     CXPLAT_THREAD_CONFIG Config = { 0, 0, "TcpPerfWorker", WorkerThread, this };
+
+    this->ExecutionContext.Callback = DoWork;
+    this->ExecutionContext.Context = this;
+    this->ExecutionContext.Ready = TRUE;
```

Comment:
Not need to use `this->` unless there is a name conflict.
```suggestion
    ExecutionContext.Callback = DoWork;
    ExecutionContext.Context = this;
    ExecutionContext.Ready = TRUE;
```